test-clients/scripts/insight.py

```diff
@@ -31,6 +31,7 @@
 
 from datetime import datetime, timedelta, timezone
 
+
```

Comment:
The graph is generated using `insight.py` that is on platform-regression repo. I believe the changes should be made there.
Please add the resulting graph after this change. Also validate if this is not affecting any platform tests.